packages/extension/src/components/DaSidebar.vue

```diff
@@ -5,15 +5,17 @@
       <div class="sidebar__filter">
         <da-mode-switch class="sidebar__filter_switch"
                         first-icon="link" second-icon="hashtag"
-                        :checked="filterChecked" @toggle="toggleFilter"/>
+                        :checked="filterChecked" @toggle="toggleFilter($event); $refs.searchTags.value = ''"/>
       </div>
+
       <div class="sidebar__element sidebar__tags__search btn btn-menu"
```

Comment:
What do you think about changing the class name `sidebar__tags__search` to `sidebar__search`?